spec/API_specification/statistical_functions.md

```diff
@@ -105,6 +105,15 @@ Calculates the product of input array `x` elements.
 
     -   axis or axes along which products must be computed. By default, the product must be computed over the entire array. If a tuple of integers, products must be computed over multiple axes. Default: `None`.
 
+-   **dtype**: _Optional\[ &lt;dtype&gt; ]_
```

Comment:
This is a question not related to the PR itself, just a matter of the style consistency across all documents. Do we always use `Optional` for all kw-only arguments? For example, `keepdims` from below does not use it, but `axis` from above does.

Reply:
@leofang Thanks for the question. No, we don't always use `Optional` for keyword-only arguments. We use [`Optional`](https://docs.python.org/3/library/typing.html#typing.Optional) for those arguments whose type is the `Union[X, None]`. In which case, for `keepdims`, we would not use `Optional` as `keepdims` cannot be set to `None`, but only a boolean `True` or `False`. Hope this clarifies!

Reply:
Thanks Athan! That helps a lot!

Reply:
Of course! :)